router.lua

```diff
@@ -126,17 +126,15 @@ function Router:match(method, path, f)
 end
 
 for _,method in ipairs(HTTP_METHODS) do
-  Router[method] = function(self, path, f)     -- Router.get = function(self, path, f)
-    return self:match(method:upper(), path, f) --   return self:match('GET', path, f)
-  end                                          -- end
+  Router[method] = function(self, path, f)                                          -- Router.get = function(self, path, f)
+    self:match(method:upper(), path, function(params) return f(params, method) end) --   return self:match('GET', path, f)
```

Comment:
I meant that only the `any` would be passing `method` to the function. The rest would stay the same. So this change would not be necessary.